extensions/shared/external-media/sources/google-photos/filter-request.js

```diff
@@ -14,9 +32,35 @@ export default function getFilterRequest( filters ) {
 		query.push( 'feature=favorite' );
 	}
 
-	if ( startDate || endDate ) {
-		const start = startDate ? startDate.substr( 0, 10 ) : '0000-00-00';
-		const end = endDate ? endDate.substr( 0, 10 ) : '0000-00-00';
+	if ( date ) {
+		let startDate = null;
+		let endDate = null;
```

Comment:
Would it make sense to have `endDate` default to `TODAY`? It looks like it'll be overwritten ever other case where that would not be desired.